support/support_test.go

```diff
@@ -140,6 +143,20 @@ func assertSanitizedDeploymentConfig(t *testing.T, dc *codersdk.DeploymentConfig
 	}
 }
 
+func assertSanitizedWorkspace(t *testing.T, ws codersdk.Workspace) {
+	t.Helper()
+	for _, res := range ws.LatestBuild.Resources {
+		for _, agt := range res.Agents {
+			for k, v := range agt.EnvironmentVariables {
+				kl := strings.ToLower(k)
+				if strings.Contains(kl, "secret") || strings.Contains(kl, "token") || strings.Contains(kl, "pass") {
```

Comment:
I'm curious if this shouldn't be configurable, so that admins can add more filters